sdk/scheduler/src/main/java/com/mesosphere/sdk/curator/CuratorUtils.java

```diff
@@ -20,14 +20,14 @@
 
     private static final String PATH_DELIM = "/";
 
-    static final int DEFAULT_CURATOR_POLL_DELAY_MS = 1000;
-    static final int DEFAULT_CURATOR_MAX_RETRIES = 3;
+    public static final int DEFAULT_CURATOR_POLL_DELAY_MS = 1000;
```

Comment:
It's just that the constants are being used in the same way in three places, could just have those places call a function instead.